contents/blog/missing-recordings.md

```diff
@@ -0,0 +1,73 @@
+---
+date: 2021-06-10
+title: The mystery of missing recordings at PostHog
```

Comment:
```suggestion
title: How we fixed a major issue with our Session Recording beta
```

Reply:
totally subjective (feel free to ignore) flyby: I think the former sounds like it'll be a more interesting read

Reply:
On reflection, you're totally the right